bread/static/js/main.js

```diff
@@ -181,3 +181,48 @@ function getBreadCookie(key, _default=null) {
     ret = ret.split('=')[1];
     return ret ? decodeURIComponent(ret) : _default;
 }
+
```

Comment:
Can you insert some comments here, to explain what the purpose of the code is?

Reply:
OK. Done.